src/features/utils/instrument-base.js

```diff
@@ -52,36 +51,45 @@ export class InstrumentBase extends FeatureBase {
     this.onAggregateImported = undefined
 
     /** used in conjunction with newrelic.start() to defer harvesting in features */
-    if (agentRef.init[this.featureName].autoStart === false) this.auto = false
-    /** if the feature requires opt-in (!auto-start), it will get registered once the api has been called */
-    if (this.auto) registerDrain(agentRef.agentIdentifier, featureName)
-    else {
-      this.ee.on('manual-start-all', single(() => {
+    this.deferred = Promise.resolve()
+
+    if (!auto || agentRef.init[this.featureName].autoStart === false) {
```

Comment:
Since auto is not used (I checked entry points), just remove it. Make `autoStart` the sole determinator.